### ISSUE TITLE
notebooks are python2 only

### ISSUE THREAD

**Issue by gijzelaerr** [aymericdamien/TensorFlow-Examples#39]
which unfortunately is not mentioned in the README

**Comment by aymericdamien** [comment]
Yes, currently only python 2 is supported. I will work on making it compatible with python 3 as well, it should just require some minor changes

**Comment by stared** [comment]
@aymericdamien As I see in most cases it is as simple as adding brackets to `print`s.
